ci-operator/step-registry/cucushift/installer/check/aws/custom-ami/cucushift-installer-check-aws-custom-ami-commands.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REGION="${LEASED_RESOURCE}"
+INFRA_ID=$(jq -r '.infraID' ${SHARED_DIR}/metadata.json)
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+  export KUBECONFIG=${SHARED_DIR}/kubeconfig
+else
+  echo "No KUBECONFIG found, exit now"
+  exit 1
+fi
+
+CONFIG=${SHARED_DIR}/install-config.yaml
+if [ ! -f "${CONFIG}" ] ; then
+  echo "No install-config.yaml found, exit now"
+  exit 1
+fi
+
+
+# case "${OCP_ARCH}" in
+# amd64)
+#     ARCH="x86_64"
+#     ;;
+# arm64)
+#     ARCH="aarch64"
+#     ;;
+# *)
+#     echo "${OCP_ARCH} is not supported, exit now"
+#     exit 1
+# esac
+
+# default_ami=$(openshift-install coreos print-stream-json | jq -r --arg a $ARCH --arg r $REGION '.architectures[$a].images.aws.regions[$r].image')
+# echo "AMI (default, \"openshift-install coreos print-stream-json\"): ${default_ami}"
+
+function is_empty()
```

Comment:
Shall we also apply this to https://github.com/openshift/release/pull/52689/files#diff-530b9f2902b21112027c35bac9423b5a4cdf151596244cfe7e16a7af5cd3d00bR61 ?